### PR TITLE
save screenshots as files ;)

### PR DESCRIPTION
**Pull request by digital-flowers**
### What does this PR do?
Save stargazer screenshots as files

### Where should the reviewer start?
Run both stargazer server and on mobile and also web view

### How should this be tested?
This PR doesn't change stargazer/app (mobile client) only the web viewer and server so you can run start stargazer from this repository and run stargazer from `tenx-react-native` you will notice the `{os}-data.json` file has image path instead of the base64 encoding and a new `screenshots` directory created next to it has the actual images 
 - please make sure to do `npm install` before testing

### Screenshots / Screen capture if any
Screenshot 2019-08-07 at 9 37 20 PM
Screenshot 2019-08-07 at 9 37 28 PM

### CODE REVIEW

**Review by AudyOdi — COMMENTED** [review]
@digital-flowers are you sure this is the correct commit? I see that the {os}-data.json still saving base64

**Comment by digital-flowers** [comment]
@AudyOdi Yeah, I have tested it!

**Comment by digital-flowers** [comment]
the client still checks if it's base64 or URL and handle it and in the server, it will generate base64 only if saving the file failed. please make sure to do `npm install` before testing it, I will add this to the pull request description

**Comment by digital-flowers** [comment]
@AudyOdi If saving the image to a file is failed it should throw an error to the console, When you have time can you please test this again and provide more details about that error you get?

File: server/test/util.test.ts [hunk 1]
```diff
@@ -5,17 +5,17 @@ import { processScreenshotDataUpload } from "../src/util";
...
       os: "ios",
       photos: [],
     });
     expect(result).toMatchInlineSnapshot(`
-            Object {
-              "message": "Success!",
-              "type": "OK",
-            }
-        `);
+                  Object {
```
Comment (TikiCat7) [inline server/test/util.test.ts:14]: seems like indent is off

**Comment by AudyOdi** [comment]
@digital-flowers i cannot get the screenshot uploaded to the example folder, only the json file updated.
```
Upload received, saving 3 ios screenshots to file: stargazer-ios-data.json...
File saved successfully!
```

**Comment by digital-flowers** [comment]
We don’t use stargate anymore

**Pull request closed without merging**